lib/livebook/session.ex

```diff
@@ -782,6 +782,17 @@ defmodule Livebook.Session do
     {:noreply, state}
   end
 
+  def handle_info({:runtime_broadcast, topic, message}, state) do
+    topics = [topic | wilcard_topics(topic)]
```

Comment:
Why are we broadcasting on multiple topics? You should have multiple subscription instead of multiple broadcasts, as each broadcast means a message across node. :)

Reply:
We want to allow pushing to specific topics like `mytopic:10`, but allow subscribing to `mytopic:*`. But I see the point, will update the channel to subscribe to individual topics, rather than wildcard.